### PR TITLE
Adkernel: rxnetwork alias

### PR DESCRIPTION
**Pull request by ckbo3hrk**
Added alias for partner network to resolve an issue with multiple connected networks via same adkernel bidder.

### CODE REVIEW

File: static/bidder-info/rxnetwork.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+aliasOf: adkernel
+gvlVendorID: 14
```
Comment (SyntaxNode) [inline static/bidder-info/rxnetwork.yaml:2]: Checking to ensure the use of adkernel's TCF vendor code is appropriate within the context of the RxNetwork alias.

Comment (ckbo3hrk) [reply static/bidder-info/rxnetwork.yaml:2]: I've intentionally specified Adkernel's gvlid to claim GDPR support. I saw a similar approach in other aliased adaptors.

Comment (SyntaxNode) [reply static/bidder-info/rxnetwork.yaml:2]: There's no need to specify it here. It will be automatically picked up with the alias.

Comment (ckbo3hrk) [reply static/bidder-info/rxnetwork.yaml]: @SyntaxNode ok, updated

**Review by SyntaxNode — APPROVED** [review]

**Review by bsardo — APPROVED** [review]

**Pull request merged**